extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -158,7 +166,21 @@ void FlowFileRepository::prune_stored_flowfiles() {
   }
 
   delete it;
+}
 
+bool FlowFileRepository::ExecuteWithRetry(std::function<rocksdb::Status()> operation) {
+  int waitTime = FLOWFILE_REPOSITORY_RETRY_INTERVAL;
+  for (int i=0; i<3; ++i) {
+    auto status = operation();
+    if (status.ok()) {
+      logger_->log_trace("Rocksdb operation executed successfully");
+      return true;
+    }
+    logger_->log_error("Rocksdb operation failed: %s", status.ToString());
+    std::this_thread::sleep_for(std::chrono::milliseconds(waitTime));
+    waitTime += FLOWFILE_REPOSITORY_RETRY_INTERVAL;
```

Comment:
The suffix `RETRY_INTERVAL` suggests that we retry every so often but here we increase the delay between retries in each iteration.

Reply:
Agreed, I'm open for suggestion :)

Reply:
Either change the behavior to always wait `FLOWFILE_REPOSITORY_RETRY_INTERVAL` milliseconds instead of increasing the wait time, or change the name to something like `FLOWFILE_REPOSITORY_RETRY_INTERVAL_INCREMENTS`. I don't like the name I've suggested, but a bad name is better than a misleading one.

Reply:
Done.